action.yml

```diff
@@ -69,15 +77,43 @@ outputs:
 runs:
   using: composite
   steps:
+  - name: Compute job Python version
+    id: compute_job_python_version
+    shell: bash
+    run: |
+      ENV_PYTHON_VERSION="${{ inputs.job-python-version }}"
+      if [ "${{ inputs.python-version }}" != "" ]; then
+        ENV_PYTHON_VERSION="${{ inputs.python-version }}"
```

Comment:
I think we need to (1) error out when both are set to their non-default values and (2) issue a warning if the deprecated input is still in use.

Here's what I've done in another action to issue warnings https://github.com/pypa/gh-action-pypi-publish/commit/9b8e7336db3f96a2939a3e9fa827c62f466ca60d — this uses a GitHub Actions Runner Command that shows up in the workflow view.

Mind looking into this?

Reply:
I can work it out (there should already be a deprecation warning when the old option is used BTW, due to https://github.com/ansible-community/ansible-test-gh-action/pull/19/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R42). But before I continue we should settle on a name for the option - what about my proposal here? https://github.com/ansible-community/ansible-test-gh-action/issues/18#issuecomment-1201714480

Reply:
Okay, sounds good.

Reply:
6c6a502d2ba5999545f867e89a7e565710e77fe1 makes that it fails when both parameters are provided (with non-default values).